src/components/Buzz/Rewards/EarlyAccessRewards.tsx

```diff
@@ -0,0 +1,141 @@
+import { Line } from 'react-chartjs-2';
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  LinearScale,
+  LineElement,
+  PointElement,
+  Tooltip as ChartTooltip,
+  Colors,
+  Legend,
+} from 'chart.js';
+import dayjs from 'dayjs';
+import { trpc } from '~/utils/trpc';
+import { useBuzzDashboardStyles } from '~/components/Buzz/buzz.styles';
+import { useMemo } from 'react';
+import { Currency, StripeConnectStatus } from '@prisma/client';
+import { Paper, Stack, Title, Text } from '@mantine/core';
+import { constants } from '~/server/common/constants';
+import { CurrencyBadge } from '~/components/Currency/CurrencyBadge';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  ChartTooltip,
+  Colors,
+  Legend
+);
+
+export const EarlyAccessRewards = () => {
+  const { data: modelVersions = [], isLoading } =
+    trpc.modelVersion.earlyAccessModelVersionsOnTimeframe.useQuery({ timeframe: 14 });
+  const { data: userStripeConnect } = trpc.userStripeConnect.get.useQuery();
+
+  const { classes, theme } = useBuzzDashboardStyles();
+  const labelColor = theme.colorScheme === 'dark' ? theme.colors.gray[0] : theme.colors.dark[5];
+
+  const options = useMemo(
```

Comment:
Not sure which is it, but we should type this object
```suggestion
  const options = useMemo<LineOptions>(
```
or
```suggestion
  const options = useMemo<ChartOptions>(
```